Sources/Workspace/DefaultExtensionRunner.swift

```diff
@@ -0,0 +1,189 @@
+/*
+This source file is part of the Swift.org open source project
+
+Copyright (c) 2021 Apple Inc. and the Swift project authors
+Licensed under Apache License v2.0 with Runtime Library Exception
+
+See http://swift.org/LICENSE.txt for license information
+See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Basics
+import TSCBasic
+import TSCUtility
+import PackageGraph
+import PackageLoading   // for ManifestResourceProvider
+import PackageModel
+import SPMBuildCore
+import Foundation
+
```

Comment:
would be good to run formatter on all the new files (like this one) so at least new additions are formatted in a standard way

Reply:
Will do.  That's `swift-lint`?

Reply:
`brew install swift-format`
`swift-format  <file or directory>`

Reply:
Great, will do that for the new files.

Reply:
I've reformatted the new ones (though using `brew install swiftformat`, not `swift-format`).